Fly_Swatter/mode_control.py

```diff
@@ -10,8 +10,19 @@
 import sched
 import random
 
+class system_run_args():
```

Comment:
this is inline with python guidelines to not have too many arguments for a function; probably a even better way to do it that should be looked into